From 9d06d1c5a917fbe636f4370a22d9175673b7280a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bastien=20ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Mon, 11 Sep 2017 14:03:34 +0200
Subject: Eliminate minipass

Forwarded: no
---
 index.js     | 2 +-
 package.json | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/index.js b/index.js
index 190eef0..43a0417 100644
--- a/index.js
+++ b/index.js
@@ -2,7 +2,7 @@
 
 // Transforms a stream of TAP into a stream of result objects
 // and string comments.  Emits "results" event with summary.
-const MiniPass = require('minipass')
+const MiniPass = require('stream').PassThrough
 
 const yaml = require('js-yaml')
 const util = require('util')
diff --git a/package.json b/package.json
index b1c3bbf..5b2720c 100644
--- a/package.json
+++ b/package.json
@@ -8,8 +8,7 @@
   },
   "dependencies": {
     "events-to-array": "^1.0.1",
-    "js-yaml": "^3.2.7",
-    "minipass": "^2.2.0"
+    "js-yaml": "^3.2.7"
   },
   "devDependencies": {
     "glob": "^7.0.5",
